### ISSUE TITLE
"import-sort-cli" doesn't match files/directories starting with "+" ending up with "No files found for the given patterns" error

### ISSUE THREAD

**Issue by vladimiry** [renke/import-sort#71]
The core issue is in `micromatch` matching, used by `globby` internally, see playground here https://codesandbox.io/s/10p9160v7l

So a possible solution is to escape "+" character, probably not only the plus one. Ie escaping each pattern before calling `globby.sync` [here](https://github.com/renke/import-sort/blob/a2daa0e94477002c84e2742068706e2d2fb4a798/packages/import-sort-cli/src/index.ts#L63).

**Comment by nfantone** [comment]
Similar issue if feeding files with `[` or `]` in their names. Pretty common nowadays with the likes of Next.js using patterns in file names.

```sh
❯ ./node_modules/.bin/import-sort --write '/path/src/pages/posts/[slug].js'
No files found for the given patterns: /path/src/pages/posts/[slug].js, !**/node_modules/**, !./node_modules/**
```

**Comment by Yedrup** [comment]
@nfantone hi, did you find a solution by chance?

**Comment by nfantone** [comment]
@Yedrup Yeah - I had opened #127 for this, but went unnoticed. I also posted [a workaround]((https://github.com/renke/import-sort/pull/127#issuecomment-680169999)) in there.
